### PR TITLE
Replacing the Developer Projects API with the latest Cloud Resource Manager API

### PR DESCRIPTION
**Pull request by meltsufin**
#496 

Developer Projects API is deprecated now, and this update upgrades the project to the latest [Cloud Resource Manager API](https://cloud.google.com/resource-manager/)

We use this API to fetch the list of projects.

### CODE REVIEW

File: google-cloud-tools-plugin/build.gradle [hunk 1]
```diff
@@ -38,7 +38,7 @@ dependencies {
     compile(project(':common-lib'))
 
     compile files('lib/google-api-services-clouddebugger-v2-rev4-1.20.0.jar')
-    compile files('lib/google-api-services-developerprojects-v1-rev20140815212553-1.19.0.jar')
+    compile files('lib/google-api-services-cloudresourcemanager-v1beta1-rev12-1.21.0.jar')
```
Comment (elharo) [inline google-cloud-tools-plugin/build.gradle]: any chance they'll put this in maven central?

**Comment by elharo** [comment]
LGTM, squash and commit

File: google-cloud-tools-plugin/build.gradle (hunk 1 above)
Comment (meltsufin) [inline google-cloud-tools-plugin/build.gradle]: I actually now see they have version v1-rev2-1.21.0 in Maven Central. Should I try using that?

**Comment by patflynn** [comment]
For now the workflow is to squash branch commits to just one commit before
merging.

**Comment by loosebazooka** [comment]
we gotta have a version of this CL in the android studio code so they can continue to work.

File: google-cloud-tools-plugin/build.gradle (hunk 1 above)
Comment (elharo) [inline google-cloud-tools-plugin/build.gradle]: yes, we'd rather not bundle the jar if we can avoid it.

Comment (patflynn) [inline google-cloud-tools-plugin/build.gradle]: +1 please use maven central artifact.

**Comment by etanshaul** [comment]
just noting that I think this is a dup of #463 (which hasn't been worked yet) so feel free to close that too once this is merged.

File: google-cloud-tools-plugin/build.gradle (hunk 1 above)
Comment (meltsufin) [inline google-cloud-tools-plugin/build.gradle]: I tried quickly, and the version in Maven Central didn't work out of the box. Need to look more into it...

**Comment by meltsufin** [comment]
@etanshaul I think we'll handle the switch to Maven Central for cloudebugger and cloudresourcemanager separately as proposed by #463, #223, and #516.

**Pull request merged**